index.html

```diff
@@ -61,5 +61,313 @@ <h2 class="supporting-text">
                 </li>
             </ul>
         </section>
+        <section class="projects">
+            <h2>
+                My recent Works
+            </h2>
+            <section class="first-project">
+                <img src="img\Project\Img Placeholder.png" alt="project one interface">
+                <h3>
+                    Multi-Post Stories
+                </h3>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                    has been the industry's standard dummy text ever since the 1500s, 
+                    when an unknown printer took a standard dummy text.
+                </p>
+                <ul>
+                    <li>
+                        <a href="#" >css</a>
+                    </li>
+                    <li>
+                        <a href="#" >html</a>
+                    </li>
+                    <li>
+                        <a href="#" >bootstrap</a>
+                    </li>
+                    <li>
+                        <a href="#" >Ruby</a>
+                    </li>
+                </ul>
+                <button>
+                    See Project
+                </button>
```

Comment:
- [ ] Your button does not have any type, I would suggest you add one of type button for all of your `see project`'s buttons